openjdk-11.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: openjdk-11
   version: 11.0.20.2
-  epoch: 2
+  epoch: 3
```

Comment:
I am bumping this here to save a step, since I messed up [here](https://github.com/wolfi-dev/os/pull/2092/files) and didn't bump it properly ([ref](https://github.com/wolfi-dev/os/commit/26ac55279ed63845ed235ea6bb31dcea54897862)). I still don't really know what went wrong with that epoch bump, but this seems like the easiest path forward.